qualtran/bloqs/chemistry/sparse/prepare.py

```diff
@@ -178,9 +182,8 @@ class PrepareSparse(PrepareOracle):
     one_body: Tuple[int, ...] = attrs.field(repr=False)
     keep: Tuple[int, ...] = attrs.field(repr=False)
     num_bits_rot_aa: int = 8
-    is_adjoint: bool = False
     sum_of_l1_coeffs: SymbolicFloat = 0.0
-    qroam_block_size: Optional[int] = None
+    qroam_block_size: int = 1
```

Comment:
Should we just store the `log_block_size` and change `block_size` to be a computed property? It makes more sense to store the log_block_size to avoid potential bugs where `block_size` is not a power of 2.

Reply:
Done.